cmd/crc-embedder/cmd/embed.go

```diff
@@ -86,19 +86,19 @@ func embedFiles(binary string, filenames []string) error {
 
 var (
 	dataFileUrls = map[string][]string{
-		"darwin": []string{
+		"darwin": {
```

Comment:
This bit is not mentioned in the commit log?

Reply:
`make fmtcheck` was not complaining on this but golangci-lint yes. I will add a note in the commit message.